test/v200/utils/common/command_test_utils.go

```diff
@@ -0,0 +1,212 @@
+package common
+
+import (
+	"fmt"
+
+	schema "github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
+)
+
+// commandAdded adds a new command to the test schema data and notifies the follower
+func (testDevFile *TestDevfile) commandAdded(command schema.Command) {
+	LogInfoMessage(fmt.Sprintf("command added Id: %s", command.Id))
+	testDevFile.SchemaDevFile.Commands = append(testDevFile.SchemaDevFile.Commands, command)
+	if testDevFile.Follower != nil {
+		testDevFile.Follower.AddCommand(command)
```

Comment:
Are you going to add the implementation for follower's `AddCommand` later?

Also I would like more description as comments or better func names. We have

`(testDevFile *TestDevfile) commandAdded()`
`Follower's AddCommand(schema.Command)`
`(testDevFile *TestDevfile) AddCommand`

IMO this is way too many "add command" functions(same for component) to keep track of. Its not easy for others reading these tests to keep track.

Reply:
I would also consider about the commandUpdated and UpdateCommand functions

Reply:
I'll take this as a to do, but be honest the function names fit. Suggestions accepted.